src/hooks/useTopVoters.ts

```diff
@@ -2,16 +2,16 @@ import { useQuery } from '@tanstack/react-query'
 
 import { Governance } from '../clients/Governance'
 
-import { FIVE_MINUTES_MS } from './constants'
+import { ONE_DAY_MS } from './constants'
 import { useSortingByKey } from './useSortingByKey'
 
-export default function useTopVoters(start: Date, end: Date, limit?: number) {
+export default function useTopVoters(limit?: number) {
```

Comment:
Limit is not necessary, right? We are not using it.

Reply:
Yeah, it's an optional parameter for the endpoint. I'll remove it from the hook